libfreerdp/core/freerdp.c

```diff
@@ -430,6 +431,22 @@ BOOL freerdp_check_event_handles(rdpContext* context)
 		return FALSE;
 	}
 
+	/* Resend mouse cursor position to prevent session lock in prevent-session-lock mode */
```

Comment:
better move this to a `static` function to encapsulate this (easier to guess what is going on than in one big function)